src/App.scss

```diff
@@ -0,0 +1,111 @@
+.Landing {
+  display: flex;
+  flex: 1;
+  flex-wrap: wrap;
+  height: 100vh;
+}
+
+.Login {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 11px;
```

Comment:
I see you using the property font size a lot. Is there a way you could define some standards for the project?
for example:
all h1's have this properties,
all p's have this other properties
And so on and so forth so you don't have to be setting font sizes this often.